### PR TITLE
Use a Trello board to manage conference process

### PR DESCRIPTION
**Pull request by croaky**
* Move instructions into the Trello board.
* Remove Dan from the process.
  Everyone uses their own judgement and the actions of their peers
  to regulate and communicate.

### CODE REVIEW

**Comment by BlakeWilliams** [comment]
I think this is a good change. I definitely prefer trello over the spreadsheet because I can easily see what conferences are upcoming and find the conference I'm looking for. I also like how it defers to our judgement instead of going through a single source. 

Maybe mention in the instructions that if you're questioning wether you should attend or not to email your MD for guidance?

**Comment by cpytel** [comment]
I have a conference I'm planning to attend. The Trello board made sense to me because a lot of what we do is in Trello so it's nice to have it in the same place, but also there *is* a workflow associated with conferences, and so it makes sense in Trello.

As someone who wants to attend a conference, it'd be nice to have this squared away so that I can feel confident with it.

I've also noticed that people are posting conferences to basecamp, but I don't see where thats documented anywhere. The conference I'm thinking of attending is an Apple conference and so we talked about it in #xcoders slack channel today and I don't think it's really necessary to post to Basecamp.

File: benefits/conferences.md [hunk 1]
```diff
@@ -2,22 +2,23 @@
...
 
-Please sign-up to attend conferences on the appropriate [conference spreadsheet][conference-spreadsheet]. If you are the first to sign-up, please add a sheet for the conference. Always ping your office manager once you have added your name to the conference spreadsheet. 
+Follow the instructions on the [Conferences Trello board][trello].
+
+[trello]: https://trello.com/b/ja0PVibp/conferences
 
 Your office manager will book your travel & purchase your ticket.
 
-If you are signing-up for a conference that sells out quickly, please purchase the ticket yourself on your company card, email the receipt to reimbursements & note on the sign-up spreadsheet that the ticket has already been purchased.
+If you are signing-up for a conference that sells out quickly,
```
Comment (mxie) [inline benefits/conferences.md]: Can we un-hyphenate "signing-up"?

**Comment by mxie** [comment]
I like this idea and the fact that I can see all the different conferences that people are interested in all at once. Scrolling through the tabs on the spreadsheets becomes a bit of a nuisance over time.

The concern I have is hearing about the conferences. Will we still do Basecamp messages (though unofficial/undocumented)? I suppose we could just subscribe to the board, but it might be too much noise since it'll send notifications for any activity on any card.

**Comment by cpytel** [comment]
You can subscribe to individual columns on a Trello board, so you can just subscribe to a new column to see things when they are added.

File: benefits/conferences.md (hunk 1 above)
Comment (croaky) [inline benefits/conferences.md]: @mxie Good call. Done in 1ecb0a2.

**Comment by croaky** [comment]
> The idea of getting rid of the Basecamp aspect of this has no appeal to me whatsoever, and Trello is not a replacement for BC to me. I like that BC sends me targeted emails and has the UI of a conversation thread.

> Will we still do Basecamp messages (though unofficial/undocumented)?

This document did not mention Basecamp before this change and doesn't mention Basecamp after the change. I'd like to leave that cultural process to others to document / champion / organically figure out.

It looks like overall, the feedback is that the Trello board is a better solution than the spreadsheet and that I can remove myself as an approver. So, I'm going to squash and merge.

**Pull request merged**